src/component/attendance/Header.js

```diff
@@ -0,0 +1,43 @@
+import { Box } from '@chakra-ui/layout'
+import {React, useState} from 'react'
+import bell from '../../assets/notification-bell.svg'
+import bell2 from '../../assets/smart-bell.svg'
+import {Center, Divider, Image} from '@chakra-ui/react'
+import {ListItem,UnorderedList} from '@chakra-ui/react'
+
+
+
+const Header = () => {
+    const [state, setstate] = useState("hidden");
```

Comment:
Please use a more descriptive variable name here, like you did for the bell state